toolkit/tools/internal/safechroot/safechroot.go

```diff
@@ -140,6 +140,18 @@ func NewOverlayMountPoint(chrootDir, source, target, lowerDir, upperDir, workDir
 	return
 }
 
+func (m *MountPoint) GetSource() string {
+	return m.source
+}
+
+func (m *MountPoint) GetFSType() string {
+	return m.fstype
+}
+
+func (m *MountPoint) GetTarget() string {
+	return m.target
+}
+
```

Comment:
They are trivial, but please still add comments for all public methods.

Reply:
fixed